learning/step_by_step/your_first_game.rst

```diff
@@ -0,0 +1,768 @@
+.. _doc_your_first_game:
+
+Your First Game
+===============
+
+Overview
+--------
+
+This tutorial will guide you through making your first Godot Engine
+project. You will learn how the Godot Engine editor works, how a project
+is structured, and how to build a 2D game.
+
+.. note:: This project is intended to introduce the Godot Engine. It 
+          assumes that you have some programming experience already. If 
+          you're new to programming entirely, you should start here:
+          :ref:`doc_scripting`.
+
+The game is called *"Dodge the Creeps"*. Your character must move and
+avoid the enemies for as long as possible. Here is a preview of the
+final result:
+
+.. image:: /img/dodge_preview.gif
+
+**Why 2D?** 
+    Beginners are urged to focus on 2D projects until they
+    have a good understanding of the game development process. 3D games
+    are significantly more complex, so stick to 2D until you feel
+    confident in your ability.
+
+Project Setup
+-------------
+
+Start by launching Godot and making a new project. Then, download
+:download:`dodge_assets.zip </files/dodge_assets.zip>` - the images and sounds you'll be
+using to make the game. Unzip these files in your new project folder.
+
+.. note:: For this tutorial, we will assume you are already familiar with the
+          Godot Engine editor window. If you haven't read :ref:`doc_scenes_and_nodes`, do so now.
+
+This game will use "portrait" mode, so we need to adjust the size of the
+game window. Click on Project -> Project Settings -> Display -> Window and
+set ``Width`` to ``480`` and ``Height`` to ``720``.
+
+Organizing the Project
+~~~~~~~~~~~~~~~~~~~~~~
+
+In this project, we will be making 3 independent scenes: ``Player``,
+``Mob``, and ``HUD``, which we will combine into the game's ``Main``
+scene. In a larger project, it might be useful to make folders to hold
+the various scenes and their scripts, but for this relatively small
+game, you can save your scenes and scripts in the root folder, which is
+referred to as ``res://``.  You can see your project folders in the Filesystem 
+Dock in the upper left corner:
+
+.. image:: /img/filesystem_dock.gif
+
+Player Scene
+------------
+
+The first scene we'll build will define the "Player" object. One of the benefits
+of creating a separate Player scene will be that we can test it separately, even
+before we've created the other parts of the game.
+
+Node Structure
+~~~~~~~~~~~~~~
+
+To begin, click the "Add/Create a New Node" button and add an :ref:`Area2D <class_Area2D>`
+node to the scene.
+
+.. image:: /img/add_node.png
+
+We are using ``Area2D`` so that we can detect other objects overlapping
+(i.e. running into) the player. Change its name to ``Player`` - this will be
```

Comment:
Avoid parens and i.e. Avoid the dash to split a sentence in 2, it's like parens.
`With ``Area2D``, we can detect other objects that collide, or run into the player. Change its name to ``Player``. This will be our scene's root or top-level node.`